server/src/ViperProtocol.ts

```diff
@@ -691,6 +693,42 @@ export class Common {
         }
     }
 
+    public static spawn(

```

Comment:
ViperProtocol.ts should have the same contents in client and server.

Reply:
Good point. I've now copy pasted the files and discovered a few more changes that have been missed in the past. However, at least now the files are in sync again.